lib/grid.rb

```diff
@@ -2,28 +2,20 @@
 class Grid
   attr_accessor :status_line
 
-  AXE_LETTERS = %w( A B C D E F G H I J ).freeze
-  AXE_DIGGITS = %w( 1 2 3 4 5 6 7 8 9 10 ).freeze
-  HIT_CHAR = 'X'.freeze
-  MISS_CHAR = '-'.freeze
-
-  def initialize
-    @matrix = []
-    @fleet = []
-  end
+  AXE_LETTERS = %w( A B C D E F G H I J )
+  AXE_DIGITS = %w( 1 2 3 4 5 6 7 8 9 10 )
+  HIT_CHAR = 'X'
+  MISS_CHAR = '-'
 
-  def build(matrix, fleet = nil)
+  def initialize(matrix, fleet = nil)
     @matrix = matrix
     @fleet = fleet
-    self
   end
 
   def show
     print_header
     setup_with_fleet if @fleet
-    @matrix.each_with_index do |grow, index|
-      Grid.row("#{AXE_LETTERS[index]} #{grow.join(' ')}")
-    end
+    @matrix.each_with_index {|grow, index| Grid.row("#{AXE_LETTERS[index]} #{grow.join(' ')}")}
```

Comment:
Yes, it could be in one line but for me it is harder to read now. I think previous version was more readable.